### DISCUSSION TITLE
Checkbox Funktion "Bei Inventur aufgefunden"

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#14] [Shipped - Live on production]
Aktuell kann man die Checkbox anklicken und dann ein Datum eintragen. Das ist mehr oder weniger einmalig. Danach kann man zwar das Datum ändern, die Checkbox bleibt aber immer aktiv bis man sie wieder rausnimmt.

Überlegung:
Einen Button anstatt einer Checkbox der automatisch das Tagesdatum (editierbar) setzt

**Comment by YvesR** [comment]
New release provides a inventory audit feature inside the device action menu and the landing page.
Device will have a inventory history and current fields show the "latest" one.
